src/CommandingOfficers/AW2/AW2Commander.java

```diff
@@ -0,0 +1,32 @@
+package CommandingOfficers.AW2;
+
+import CommandingOfficers.Commander;
+import CommandingOfficers.CommanderInfo;
+import CommandingOfficers.TrilogyCommander;
+import CommandingOfficers.CommanderInfo.InfoPage;
+import Engine.GameScenario;
+
+public abstract class AW2Commander extends TrilogyCommander
+{
+  private static final long serialVersionUID = 1L;
+  public static final InfoPage AW2_MECHANICS_BLURB = new InfoPage(
+      "Gain +10 defense on activating a power.\n"
```

Comment:
It's strange to have this separate from the advertised power stats. Like, is it whatever the power claims, plus an additional 10 defense? And I guess now we might have to ask what kind of defense it's adding, too.

Reply:
OTOH, it's kind of ugly to glue that on every power description ever (Reboot Camp does this, FWIW).
Yup. +10/10 for AW3, x1.1/0.9 for AW1 (Did I mention it's special?)

Non-DoR gives subtractive defense, unless stated otherwise.
I guess I could blurb everything, but bleeeeeeh

Reply:
Sounds like Reboot Camp learned that sometimes ugly/repetitive/verbose == clear. So I think I agree with them, and I'd co-locate the power stuff, but you are the one doing the work, so \_( ' ' )_/.